cmd/plume/types.go

```diff
@@ -78,3 +78,21 @@ type channelSpec struct {
 	Azure        azureSpec
 	AWS          awsSpec
 }
+
+type fcosChannelSpec struct {
+	Bucket  string
+	Profile string
+	Region  string
+}
+
+type ReleaseMetadata struct {
+	Observation string          `json:"observation"` // used to note to users not to consume the release metadata index
+	Releases    []BuildMetadata `json:"releases"`
+	Timestamp   string          `json:"timestamp"`
```

Comment:
Probably going to change to:
```json
{
    "metadata": {
        "last-modified": "<timestamp>"
    }
}
```

per https://github.com/coreos/fedora-coreos-tracker/issues/98#issuecomment-499291779